starboard/nplb/time_zone_get_current_test.cc

```diff
@@ -34,9 +36,23 @@ TEST(SbTimeZoneGetCurrentTest, IsKindOfSane) {
   // ... and +24 hours from the Prime Meridian, inclusive
   EXPECT_LE(zone, 24 * 60);
 
-  if (zone == 0) {
-    SB_LOG(WARNING) << "SbTimeZoneGetCurrent() returns 0. This is only correct "
-                       "if the current time zone is the same as UTC";
+  static auto const* time_zone_extension =
+      static_cast<const StarboardExtensionTimeZoneApi*>(
+          SbSystemGetExtension(kStarboardExtensionTimeZoneName));
+  if (time_zone_extension) {
+    ASSERT_STREQ(time_zone_extension->name, kStarboardExtensionTimeZoneName);
+    ASSERT_EQ(time_zone_extension->version, 1u);
+    time_zone_extension->SetTimeZone("UTC");
+    zone = SbTimeZoneGetCurrent();
+    EXPECT_EQ(zone, 0);
+
+    time_zone_extension->SetTimeZone("US/Pacific");
+    zone = SbTimeZoneGetCurrent();
+    EXPECT_EQ(zone, 480);
```

Comment:
These values don't account for daylight savings, do they? I think the offset would be different based on the current time.

Reply:
They don't account for daylight saving. At least based on the comments for SbTimeZoneGetCurrent()

`// The number of minutes west of the Greenwich Prime Meridian, NOT including
// Daylight Savings Time adjustments.`

The daylight saving adjustment is being applied by ICU later.